{{ cookiecutter.app_name }}/{{ cookiecutter.app_name }}/config.py

```diff
@@ -0,0 +1,25 @@
+import os
+from typing import Literal, Optional
+
+from pydantic import BaseSettings, HttpUrl
+
+app_env = os.environ.get("APP_ENV", "dev")
+
+
+class Settings(BaseSettings):
```

Comment:
To avoid constantly loading .env file
```suggestion
@functools.lru_cache
class Settings(BaseSettings):
```

Reply:
added

Reply:
okay, we can add later if needed